Dockerfile

```
@@ -3,8 +3,8 @@ FROM registry.access.redhat.com/ubi8/ubi:8.3 AS manifest
 COPY .git /tmp/.git
 
 RUN cd /tmp && \
-    head=$(cat .git/HEAD |cut -d " " -f 2) && \
-    sha=$(cat .git/$head) && \
+    sha=$(cat .git/HEAD |cut -d " " -f 2) && \
+    if [[ $(cat .git/HEAD) == ref* ]];then sha=$(cat .git/$sha); fi && \
```

Super-minor

```suggestion
    sha=$(cat .git/HEAD |cut -d " " -f 2) && \
    if [ $(cat .git/HEAD) == ref* ]; then sha=$(cat .git/$sha); fi && \
```

---

Also wondering if we should enquote things for safety

```suggestion
    sha=$(cat .git/HEAD |cut -d " " -f 2) && \
    if [[ "$(cat .git/HEAD)" == "ref:"* ]];then sha=$(cat .git/$sha); fi && \
```

---

Oh ignore the [[]] to [] change...didn't realize that * required the [[]]